billkabanga/fast-food-fast-react
# #163722478: Users should be able to signup

**What does this PR do?**
Implement signup functionality for users to be able to signup into the fast food fast application.

**Description of tasks to be completed?**
- Modify Signup container component with event handlers.
- Modify the signup component with reusable elements.

**How can this be manually tested?**
- clone the repository.
- Check out this branch `ft-signup-163722478`
- Run the app using `yarn start`
- Navigate to the signup page on this route `/signup`

**What are the related pivotal tracker stories?**
#163722478

**Screenshots?**
![image](https://user-images.githubusercontent.com/40595048/52357384-226b1b00-2a47-11e9-9fa7-fb00e018947c.png)
